blog/index.html

```diff
@@ -26,6 +26,9 @@ <h1 itemprop="headline"><a href="{{ post.url }}">{{ post.title }}</a></h1>
                   {{ post.date | date_to_string }}.
                 </time>
               </p>
+              <p class="authors">
+                by {% for author in post.authors %}{{author | lookup:"authors, full_name"}}{% unless forloop.last %}, {%endunless%}{% endfor %}
```

Comment:
Fixed this, moved the whole thing into an include in case there are other permutations of the logic (though I don't suspect there will be any time soon)